graphql/src/main/java/com/scalar/db/graphql/schema/CommonSchema.java

```diff
@@ -0,0 +1,112 @@
+package com.scalar.db.graphql.schema;
+
+import static graphql.schema.GraphQLEnumType.newEnum;
+import static graphql.schema.GraphQLInputObjectField.newInputObjectField;
+import static graphql.schema.GraphQLInputObjectType.newInputObject;
+import static graphql.schema.GraphQLList.list;
+import static graphql.schema.GraphQLNonNull.nonNull;
+
+import com.google.common.collect.ImmutableSet;
+import com.scalar.db.api.ConditionalExpression;
+import com.scalar.db.api.Consistency;
+import com.scalar.db.api.Scan;
+import graphql.Scalars;
+import graphql.introspection.Introspection.DirectiveLocation;
+import graphql.schema.GraphQLArgument;
+import graphql.schema.GraphQLDirective;
+import graphql.schema.GraphQLInputObjectType;
+import graphql.schema.GraphQLNamedInputType;
+import java.util.Set;
+
+public class CommonSchema {
+  public static Set<GraphQLNamedInputType> createCommonGraphQLTypes() {
+    GraphQLInputObjectType conditionalExpressionInputObject =
+        newInputObject()
+            .name("ConditionalExpression")
+            .field(newInputObjectField().name("name").type(nonNull(Scalars.GraphQLString)))
+            .field(newInputObjectField().name("intValue").type(Scalars.GraphQLInt))
+            .field(newInputObjectField().name("floatValue").type(Scalars.GraphQLFloat))
+            .field(newInputObjectField().name("stringValue").type(Scalars.GraphQLString))
+            .field(newInputObjectField().name("booleanValue").type(Scalars.GraphQLBoolean))
```

Comment:
Do we need to add `bigIntValue` here?

Reply:
Thank you!
I added the `bigIntValue` with the type `GraphQLFloat` (double precision IEEE 754) for this PR. But I’m going to implement a custom GraphQL scalar type that exactly matches the range of `BigIntValue` (-2^53 - 2^53) in another PR.